Tests/Grid/Mapping/SourceTest.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+namespace APY\DataGridBundle\Grid\Tests\Mapping;
+
+use APY\DataGridBundle\Grid\Mapping\Source;
+use PHPUnit\Framework\TestCase;
+
+class SourceTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->source = new Source([]);
+    }
+
+    public function testSetterColumnsWithDefaultValue()
```

Comment:
You're not testing a `setter`, so `testColumnsHasDefaultValue` is better (this is valid also for all the tests below)